staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -678,6 +686,16 @@ message NetworkUsage {
     repeated NetworkInterfaceUsage interfaces = 3;
 }
 
+// WindowsNetworkUsage contains data about network resources specific to Windows.
+message WindowsNetworkUsage {
+    // The time at which these stats were updated.
```

Comment:
Nit: Keep the timestamp comment consistent 

```
Timestamp in nanoseconds at which the information were collected. Must be > 0.
```

Reply:
This was from linux network usage comment above, I'll updated both to be consistent.